src/inputs/currency-input.jsx

```diff
@@ -101,10 +101,13 @@ class CurrencyInput extends React.Component {
         onBlur={this.onFocusOut}
       >
         <span className={labelTextClasses}>
-          {label}
+          <React.Fragment>{label}</React.Fragment>
           {attr.required && (
-            <span>
-              {'\u00A0'}*<span className='u-sr-only'> required field</span>
+            <span key='u-sr-only'>
+              {'\u00A0'}*<span key='u-sr-only' className='u-sr-only'>
```

Comment:
The outer span should have a descriptive key like `required-helper-text` or something.

The inner span doesn't need a key.

Reply:
ooops, thought I deleted those